packages/sql-mapper/lib/entity.js

```diff
@@ -205,8 +205,12 @@ function createMapper (db, sql, log, table, fields, primaryKey, relations, queri
       query = sql`${query} ORDER BY ${sql.join(orderBy, sql`, `)}`
     }
 
-    if (opts.limit && opts.offset !== undefined) {
-      query = sql`${query} LIMIT ${opts.limit} OFFSET ${opts.offset}`
+    if (opts.limit || opts.offset !== undefined) {
+      const limit = (opts.limit !== undefined) ? opts.limit : Number.MAX_SAFE_INTEGER
```

Comment:
`Number.MAX_SAFE_INTEGER` is the default value for `limit` parameter when there is `offset` parameter in the request, because in the sql query you cannot use `OFFSET` keyword without `LIMIT`

Reply:
Can you move this explanation in a commment in the code?

Reply:
Sure, done.